pisa/stages/utils/pi_kde.py

```diff
@@ -0,0 +1,117 @@
+
+"""
+Stage to transform arrays with weights into KDE maps
+that represent event counts
+"""
+from __future__ import absolute_import, print_function, division
+import numpy as np
+
+from pisa import FTYPE, TARGET
+from pisa.core.pi_stage import PiStage
+from pisa.utils.log import logging
+from pisa.utils.profiler import profile
+from pisa.utils import vectorizer
+from pisa.utils.kde_hist import kde_histogramdd
+
+class pi_kde(PiStage):
+    """
+    stage to kde map events
+
+    Paramaters
```

Comment:
sp

Reply:
Ah crap, that file is in an separate PR....but now i accidentally committed it here too. Should we do that all in one PR now or want me to remove this file? (can i `git rm`?)

Reply:
Nah, just fix this spelling mistake that other branch, merge that other PR/branch to PISA, merge the  `i3os/master` back into this branch, and then I don't think this file will show up with this PR any longer. Then we can "separate concerns," so-to-speak :wink: ...